### ISSUE TITLE
MonetaryValue of the lot is not linked to Lot, in case of a single lot

### ISSUE THREAD

**Issue by csnyulas** [OP-TED/ted-rdf-mapping#186]
From Natalie: Cannot find a value in 661127 and this has just one lot.

**Comment by csnyulas** [comment]
It is related to the fix of issue #136 (which works for many lots, but not for a single lot)

**Comment by csnyulas** [comment]
This SPARQL query demonstrates that all `epo:MonetaryValue` instances are referenced (i.e. there is no MonetaryValue instance that has no property pointing to it).
```
SELECT ?type ?s ?p ?mv
{
 ?mv a epo:MonetaryValue .
 OPTIONAL { 
  ?s ?p ?mv.
  OPTIONAL { ?s a ?type.} 
 }
}
ORDER BY ?type
```

A somewhat more advanced query shows that some of the Lots are part of a procedure by themselves, while others as one of many lots belonging to the same procedure.
```
SELECT ?type ?s ?proc ?p ?mv
{
 ?mv a epo:MonetaryValue .
 OPTIONAL { 
  ?s ?p ?mv.
  OPTIONAL { ?s a ?type.} 
  OPTIONAL { ?proc epo:hasProcurementScopeDividedIntoLot ?s }
 }
}
ORDER BY ?type
```
